e3db/types/note.py

```diff
@@ -390,9 +374,9 @@ def get_expiration(self) -> str:
         expiration : str
             RFC3339 String 
         """
-        return self.__expiration
+        return self.__note_options.expiration
```

Comment:
No change to make here because 
1.) I don't get the value of all of these getters when the caller can access the note_options for a given note directly
2.) It's actually nicer for future refactoring if note obscures the fact that there is some other class / object managing these fields, so if in the future we consolidate or refactor note options / note keys we only have to change it in one place
3.) Note the conflict between 1 and 2

But useful to think about: https://en.wikipedia.org/wiki/Law_of_Demeter